piker/brokers/ib.py

```diff
@@ -328,8 +329,9 @@ async def find_contract(
             exch = 'SMART' if not exch else exch
             contract = (await self.ib.qualifyContractsAsync(con))[0]
 
-            head = await self.get_head_time(contract)
-            print(head)
+            # head = await self.get_head_time(contract)
```

Comment:
turns out we don't really need this unless looking up data to epoch - but even in that case pretty sure we'll just not receive datums past it.